min-turnup/phase2/ansible/playbooks/roles/master/templates/kubeconfig.jsonnet

```diff
@@ -1,26 +0,0 @@
-function(cfg)
```

Comment:
Oh, it looks like you had it in both `master`/`node` and I removed the redundant duplicate from `master`.